charts/signoz/values.yaml

```diff
@@ -1403,6 +1403,41 @@ otelCollector:
           filesystem: {}
           network: {}
     processors:
+      # default parsing of logs
+      logstransform/internal:
```

Comment:
This processor will be getting deprecated.

Reply:
@srikanthccv This processor is just experiemental  https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/logstransformprocessor .

As of now, there is no other way you can use the log parsing operator without it. Did you find something anywhere where it confirms it will be deprecated?

Reply:
Not confirmed, there was some discussion in the contrib repo. I think there is non-trivial feature gap b/w logstransform and transform which is why it isn't deprecated yet. I will share the issue.

Reply:
Yeah the transform is totally different, logstransform relies on stanza operators while transform doesn't

Reply:
This is the issue I was referring to open-telemetry/opentelemetry-collector-contrib/issues/19775

Reply:
Right, but as of the now the transform processor doesn't support all the operators. Which will definately be added in the future. But until then we will have to use this.